contrib/ci/test-swift.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -eof pipefail
+
+TEST_ROOT=$(dirname "${BASH_SOURCE}")/
+source "${TEST_ROOT}/test.sh"
+
+# make sure we are in this dir
+CURRENT_DIR=$(cd $(dirname $0); pwd)
+
+create-postgres-creds
+
+puts-step "fetching openstack credentials"
+
+# turn creds into something that we can use.
+mkdir -p $CURRENT_DIR/tmp/swift
+
+# guess which value to use for tenant:
+TENANT=""
+
+echo "test:tester" > $CURRENT_DIR/tmp/swift/username
+echo "testing" > $CURRENT_DIR/tmp/swift/password
+echo ${TENANT} > $CURRENT_DIR/tmp/swift/tenant
+echo "http://swift:8080/auth/v1.0" > $CURRENT_DIR/tmp/swift/authurl
+echo "1" > $CURRENT_DIR/tmp/swift/authversion
+echo "deis-swift-test" > $CURRENT_DIR/tmp/swift/database-container
+
+# kill containers when this script exits or errors out
+trap 'kill-container $SWIFT_DATA' INT TERM
+# boot swift
+SWIFT_DATA=$(docker run -v /srv --name SWIFT_DATA busybox)
+
+# kill containers when this script exits or errors out
+trap 'kill-container $SWIFT_JOB' INT TERM
+SWIFT_JOB=$(docker run --name onlyone --hostname onlyone -d -p 12345:8080 --volumes-from SWIFT_DATA -t kmala/swift-onlyone)
```

Comment:
where is the repo for this image? Considering that it's an unknown image outside of our control, would you mind setting this image up with automated changes? I have no idea what changes you've made from https://hub.docker.com/r/morrisjobke/docker-swift-onlyone/.